src/test/java/com/ericsson/ei/flowtests/FlowTest.java

```diff
@@ -97,9 +98,10 @@ String setEventsFilePath() {
     }
 
     @Override
-    Map<String, String> setCheckInfo() {
-        Map<String, String> checkInfo = new HashMap<>();
-        checkInfo.put(AGGREGATED_OBJECT_ID, AGGREGATED_OBJECT_FILE_PATH);
-        return checkInfo;
+    Map<String, JsonNode> setCheckData() throws IOException {
```

Comment:
you do not set anything here but get the data so the name should be getExpectedData